cmd/importer/run-in-cluster/importer.yaml

```diff
@@ -12,7 +12,7 @@ spec:
     - -n=ns1,ns2
     - --queuemapping-file=/mapping.yaml
     - --dry-run=false
-    - -v
+    - -vv
```

Comment:
use `=2`. It's easier to understand and modify.

Reply:
it's not ,  however done